topology/University of Michigan/AGLT2/AGLT2.yaml

```
@@ -145,29 +145,7 @@ Resources:
         Tertiary:
           ID: a833f94ad3840fe779702409c6b778d79aaed76f
           Name: Shawn McKee
-    Description: New SL6 CE at AGLT2
-    Active: false
-    FQDN: gate04.aglt2.org
-    ID: 547
-    Services:
-      CE:
-        Description: Compute Element
-        Details:
-          hidden: false
-    VOOwnership:
-      ATLAS: 90
-    WLCGInformation:
-      APELNormalFactor: 0
-      AccountingName: US-AGLT2
-      HEPSPEC: 124200
-      InteropAccounting: true
-      InteropBDII: true
-      InteropMonitoring: true
-      KSI2KMax: 31050
-      KSI2KMin: 31050
-      StorageCapacityMax: 0
-      StorageCapacityMin: 0
-  AGLT2_SL7:
+    AGLT2_SL7:
```

```suggestion
  AGLT2_SL7:
```
Looks like the indent was broken here.